src/model/MintInfo.ts

```diff
@@ -94,4 +94,7 @@ export class MintInfo {
 	get motd() {
 		return this._mintInfo.motd;
 	}
+	get amountless() {
+		return this._mintInfo.amountless;
```

Comment:
The mint signals support via the MeltMethodOptions, not on the root level of the InfoReponse. Please check the PR again and make sure to adjust this accordingly.

Reply:
i can't seem to find `MeltMethodOptions` ,  is this something that already exist or we have to create a new one for it @Egge21M

Reply:
It is there, but it might need some refactoring for this PR. You can find it as `GetInfoResponse.nuts.5.methods`